modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -19285,6 +19285,7 @@ this-area-is-defined-by-the-theme.-you-can-change-the-theme-settings-by-clicking
 this-article-cannot-be-published-because-its-parent-has-not-been-published=This article cannot be published because its parent has not been published yet.
 this-article-is-configured-to-use-a-display-page-that-does-not-exist-on-the-current-site=This article is configured to use a display page that does not exist on the current site. For it to function properly, the page with UUID {0} must be imported, or a new display page must be selected.
 this-article-is-currently-being-edited-by-another-user=This article is currently being edited by another user. Actions like edit, expire, move, and delete are temporarily unavailable.
+this-article-is-now-under-control-of-x=This article is now under control of {0}. Please note that the last changes can not be saved.
```

Comment:
@austinchiang Can you double check this literal? It sounds weird to me.

Reply:
Thanks @austinchiang . I've done the change @adolfopa